projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html

```diff
@@ -75,19 +75,18 @@ <h1>{{ record.metadata.name }}</h1>
         <i class="fa fa-envelope-o mr-2"></i>{{ 'Notification settings' | translate }}
       </ng-template>
       <!-- NOTIFICATION SETTINGS -->
-      <article *ngIf="record.metadata.notification_settings as notificationSettings">
-        <section *ngFor="let setting of notificationSettings">
-          <h5 translate>{{ setting.type }}</h5>
-          <dl class="row mb-3 ml-2">
-            <!-- EMAIL -->
-            <dt class="col-sm-3 offset-sm-2 offset-md-0">{{ 'Email' | translate }}:</dt>
-            <dd class="col-sm-7 col-md-8 mb-0">{{ setting.email }}</dd>
-            <!-- DELAY AVAILABILITY -->
-            <dt class="col-sm-3 offset-sm-2 offset-md-0" *ngIf="setting.delay">{{ 'Delay' | translate }}</dt>
-            <dd class="col-sm-7 col-md-8 mb-0">{{ setting.delay }}</dd>
-          </dl>
-        </section>
-      </article>
+      <ul class="list-unstyled" *ngIf="record.metadata.notification_settings as notificationSettings">
+        <li class="row" *ngFor="let setting of notificationSettings">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 font-weight-bold">{{ setting.type | ucfirst | translate }}:</dt>
```

Comment:
`class="offset-2 col-3"` for "DT" tag and `class="col-7"` for "DD" tag

Reply:
done